ts/src/luno.ts

```diff
@@ -111,7 +111,7 @@ export default class luno extends Exchange {
                         'listorders': 1,
                         'listtrades': 1,
                         'orders/{id}': 1,
-                        'quotes/{id}': 1,
```

Comment:
was this removed?

Reply:
Yes, and was part of a second commit I did earlier.